### DISCUSSION TITLE
Indexing/Numbering

### DISCUSSION THREAD

**Discussion by rkretsch** [fjossinet/RNArtist#22] [Q&A]
Is there currently a way to change the numbering, ideally a scriptable way? For example if my first nucleotide is 122 instead of the default 1, I would want a "130" label on the 8th nucleotide, "140" on the 18th, etc.

**Comment by fjossinet** [comment]
This is something I need too now, so your question is a great opportunity for me to think about it ;-) Scriptable means that an addition to the DSL syntax of RNArtistCore would be enough for you at first ? You just need to see this numbering in your PNG/SVG exports produced from the command line ? Or you need to visualize this numbering in the RNArtist app?

**Reply by rkretsch** [reply]
No need to integrate it into RNArtist , an option in RNArtistCore's syntax would be great! I use RNAartist to interactively obtain reasonable layout, but in the end numbering via command line is all I need.

**Reply by fjossinet** [reply]
ok, I'm gonna work on it. I let you know when it's ready.
